theme_report/env_sample.json

```diff
@@ -1,5 +1,7 @@
 {
 	"API_URL": "",
 	"API_KEY": "",
-	"CANVAS_URL": "https://instructure.com/accounts/"
+	"ROOT_ACCOUNT": "1",
+	"DISABLE_RESOURCES": "true",
+	"HEADLESS": "false"
```

Comment:
According to the README file, this parameter should be 

"HEADLESS_BROWSER": false

NOTE the param is of boolean type, not String.

Reply:
Okay I'll update these, I left them unset so they just used the defaults when I was testing.